### PR TITLE
feat: add replit base conf

### PR DESCRIPTION
**Pull request by sfroment**
Hey :wave:,

This PR is designed to add the .replit config file so that anybody can use replit easily by importing the repo ! 
So few changes were need in:
- add a cmd to be able to run the example with vite preview 
- add a cmd in the root pkg to build the example package
- add the replit file (which as an EXAMPLE config var if people which to run smthing else than the grid example)

You can already preview on https://grid.sacha42.com

Signed-off-by: Sacha Froment <sfroment42@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
 		"proto-gen:object": "buf generate packages/object/src/proto -o packages/object/src/proto",
 		"proto-gen:network": "buf generate packages/network/src/proto -o packages/network/src/proto",
 		"release": "release-it",
-		"test": "vitest"
+		"test": "vitest",
+		"build-examples": "pnpm --filter 'ts-drp-example*' build"
```
Comment (d-roak) [inline package.json:26]:
```suggestion
		"build": "pnpm build:packages && pnpm build:examples",
		"build:packages": "pnpm --filter '@ts-drp/*' build",
		"build:examples": "pnpm --filter 'ts-drp-example*' build"
```

- add the build scripts for everything
- change the postinstall to run `pnpm build:packages`
- put it in alphabetical order

(refactor the `.replit` with this changes)

**Review by d-roak — COMMENTED** [review]
how do i run this on replit?? hehehe

**Comment by sfroment** [comment]
@d-roak do you with me to write instruction in the README.md ? I'd give you screen

**Comment by sfroment** [comment]
1. Go on replit
2. Create a repl
Screenshot 2024-12-05 at 09 42 45
3. Choose import from github
Screenshot 2024-12-05 at 09 43 48
4. From there 2 choice, either connect your github account or import from URL
5. While this isn't merge on master we'd have to discard the .replit file auto create by repl and change branch (but this step won't be necessary once the PR is merge)
Screenshot 2024-12-05 at 09 46 01
Search for git
Screenshot 2024-12-05 at 09 46 42
discard all
Screenshot 2024-12-05 at 09 47 04
fetch the branches and change to my branch
Remember all this would be unnecessary later on. Also once the official topology repl is UP people can also fork it directly like a github repo.
6. Hit run or deploy (deploy require a paid account)
Screenshot 2024-12-05 at 09 49 04

**Comment by d-roak** [comment]
can you add a `replit.md` with all that stuff?

File: package.json (hunk 1 above)
Comment (d-roak) [reply package.json]: @sfroment u missed the comment after the suggestion

**Comment by sfroment** [comment]
Will do

**Comment by sfroment** [comment]
dunnow if I've put in in the right place, nevertheless you can see preview there: https://github.com/sfroment/ts-topology/blob/dev/sfroment/replit/docs/replit.md

**Comment by d-roak** [comment]
works! Only missing this 
image

**Comment by sfroment** [comment]
:face_palm:

File: package.json (hunk 1 above)
Comment (sfroment) [reply package.json]: @d-roak shall be good ! 
for the 1st point that was the suggestions right ?

**Review by d-roak — APPROVED** [review]
lgtm

**Pull request merged**